utils/lib/rr_graph/tests/test_channel.py

```diff
@@ -2,7 +2,7 @@
 # Run `python3 -m unittest utils.lib.rr_graph.tests.test_channel`
 import unittest
 
-from lib.rr_graph.channel import Track, ChannelGrid
+from ..channel import Track, ChannelGrid
```

Comment:
Weren't we using the relative import less in the test files?

Reply:
I did switch to full imports to get tests up and running in #520. I realized that the tests directories needed a `__init__.py` and is better solution.

Reply:
I think I disagree that adding __init__.py is the right idea.  If we move to the python setup.py installation of modules, that implies this file will be "installed" too, which seems wrong.  I believe the traditional structure is:

<module>/__init__.py ...
tests
doc

etc.

I don't have a problem with the tests being located in utils/lib/rr_graph/tests, but I do think it's weird that lib/rr_graph/tests is part of the module.

Reply:
Let's solve this later.